integration/inspektor-gadget/integration_test.go

```diff
@@ -458,16 +458,34 @@ func TestDns(t *testing.T) {
 			expectedEntries := []*dnsTypes.Event{
 				{
 					Event:      BuildBaseEvent(ns),
+					Qr:         "Q",
 					Nameserver: "8.8.4.4",
 					PktType:    "OUTGOING",
 					DNSName:    "inspektor-gadget.io.",
 					QType:      "A",
 				},
 				{
 					Event:      BuildBaseEvent(ns),
+					Qr:         "R",
+					Nameserver: "8.8.4.4",
+					PktType:    "HOST",
+					DNSName:    "inspektor-gadget.io.",
+					QType:      "A",
```

Comment:
I don't know how useful it is to have a response entry without the actual list of responses. However, I suppose it is an intermediate step, right?

Maybe this link could be useful for parsing the response: https://git.kernel.org/pub/scm/network/connman/connman.git/tree/src/dnsproxy.c#n972

Reply:
I wonder if the DNS ID could be useful for users to associate a DNS response to a query. Maybe we could add it but hidden by default?

Reply:
Yes, showing the responses would be good. But I was shocked to see how the responses are encoded with offsets relative to the beginning of the header. Because of this, it cannot easily be done from userspace without access to the full DNS packet. And parsing this in eBPF seems potentially difficult with the verifier, so we can keep it for another PR.

Good idea about the DNS ID, I will add it.